narwhal/primary/src/synchronizer.rs

```diff
@@ -393,90 +411,133 @@ impl Synchronizer {
                     warn!("No consensus commit happened for {:?}, triggering certificate fetching.", FETCH_TRIGGER_TIMEOUT);
                     continue;
                 };
-                if result.is_err() {
-                    debug!("Synchronizer is shutting down.");
-                    return;
-                }
-                let _scope = monitored_scope("Synchronizer::gc_iteration");
-                let gc_round = rx_consensus_round_updates.borrow().gc_round;
-                let Some(inner) = weak_inner.upgrade() else {
+                    if result.is_err() {
```

Comment:
The change in indentation doesn't look right? In fact, I think by keeping `async move {` in the same line as `spawn_logged_monitored_task!(`, a lot of unnecessary changes can be avoided.